### PR TITLE
Add altair.app v1.5.2

### PR DESCRIPTION
**Pull request by wKovacs64**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not already refused in [closed issues]. - **Corrections made.**
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by wKovacs64** [comment]
This was previously submitted in #35443 by the project author, but was rejected for various reasons. I hope to have remedied those in this new PR.

CC @imolorhe

File: Casks/altair.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'altair' do
+  version '1.5.1'
```
Comment (commitay) [inline Casks/altair.rb]: `1.5.2` has been released
https://github.com/imolorhe/altair/tree/v1.5.2

File: Casks/altair.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  sha256 'cf67b1bb34c3cfb0c10e7d5be791bed91e7b120cec24f784950f0bd0241d799f'
+
+  # github.com/imolorhe/altair was verified as official when first introduced to the cask
+  url "https://github.com/imolorhe/altair/releases/download/v#{version}/altair-darwin-x64-#{version}.zip"
+  appcast 'https://github.com/imolorhe/altair/releases.atom', checkpoint: '8bc9bec851fff81dbe9c1e0c27108445003488516eafd9e5e5ca4cbe3de80913'
```
Comment (commitay) [inline Casks/altair.rb]: `checkpoint` should be on the next line.

Example:
https://github.com/caskroom/homebrew-cask/blob/0480e1f119f2742586f35b65a4efbfba670757b8/Casks/autodmg.rb#L5-L8

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/altair.rb (hunk 1 above)
Comment (wKovacs64) [reply Casks/altair.rb]: I saw that, but `1.5.2` is just showing a blank screen for me (bug report filed) and the latest download linked from the homepage by the author is still `1.5.1` so I went with that.

File: Casks/altair.rb (hunk 2 above)
Comment (wKovacs64) [reply Casks/altair.rb]: Fixed.

**Review by commitay — APPROVED** [review]

File: Casks/altair.rb (hunk 1 above)
Comment (imolorhe) [reply Casks/altair.rb]: @wKovacs64 Thanks for this. The build for `1.5.2` should be done in an hour or so.

**Review by imolorhe — APPROVED** [review]

**Pull request merged**